src/MusicStore.Spa/Infrastructure/BuddyModelMetadataProvider.cs

```diff
@@ -27,7 +27,7 @@ protected override CachedDataAnnotationsModelMetadata CreateMetadataPrototype(IE
                         realProperty.IsRequired = buddyProperty.IsRequired;
                         realProperty.DisplayName = buddyProperty.DisplayName;
                         realProperty.DisplayFormatString = buddyProperty.DisplayFormatString;
-                        realProperty.SimpleDisplayText = buddyProperty.SimpleDisplayText;
+                        realProperty.SimpleDisplayProperty = buddyProperty.SimpleDisplayProperty;
```

Comment:
We should just clean this up wholesale. We implemented buddy types in MVC, so this shouldn't be needed anymore.